packages/testing/package.json

```diff
@@ -13,8 +13,7 @@
   },
   "author": "open-wc",
   "homepage": "https://github.com/open-wc/open-wc/tree/master/packages/testing",
-  "module": "index.js",
-  "type": "module",
```

Comment:
why remove the `"type": "module",`?

Reply:
Removing this allows the package to be used in apps that use CommonJS w/ Node16 module resolution in TS. This same change was made to the `scoped-elements` package as part of #2454.